clang/include/clang/Basic/AttrDocs.td

```diff
@@ -7671,9 +7671,12 @@ The ``[[clang::coro_lifetimebound]]`` is a class attribute which can be applied
 to a coroutine return type (`CRT`_) (i.e.
 it should also be annotated with ``[[clang::coro_return_type]]``).
 
-All parameters of a function are considered to be lifetime bound. See `documentation`_
-of ``[[clang::lifetimebound]]`` for more details.
-if the function returns a coroutine return type (CRT) annotated with ``[[clang::coro_lifetimebound]]``.
+All parameters of a function are considered to be lifetime bound if the function returns a
```

Comment:
I have asked a person for their input regarding parameter attributes. IMO this can be added later as a followup, as the function attribute is definitely necessary for brevity (as compared to annotating all parameters).

(Keeping this open for others to comment.)